benchmark/README.md

```diff
@@ -84,7 +85,7 @@ Build steps (with example options):
 
 1. `$ mkdir build; cd build`
 2. `$ cmake [path to swift src]/benchmark -G Ninja -DSWIFT_EXEC=[path to built swiftc]`
-3. `$ ninja swift-benchmark-macosx-x86_64`
+3. `$ ninja swift-benchmark-macosx-$(uname -m)`
```

Comment:
I was going to object that `uname` isn't on Windows, but looking at the docs they all seem to just be optimized for mac development anyway, so this is fine.